### DISCUSSION TITLE
Github robot adding commit to insert link to Codespaces into README.md and causing divergent repo.

### DISCUSSION THREAD

**Discussion by profsmucker** [community/community#140803] [GitHub Education]
### Hi everyone,

I'm using GitHub classroom with a templated starter Repo that includes my own README.md.  I have Codespaces turned on for them.  My students accept their assignment and the go and create their Codespace, which clones the repo for them to the Codespace.

Then, after they have cloned the repo, the github classroom robot comes along and edits my README.md and inserts into the top of it a link to Codespaces.  This edit creates a commit on the main branch in the origin repo. 

The student's repo on Codespaces is now diverged from origin and they cannot commit and push without needing to resolve a merge conflict caused by GitHub's stupid robot.

Please stop inserting links into the README.md, or if you are going to do so, do it at the time of repo creation before students have a chance to clone.  Now I have to guide 100 students through how to merge their repos, which was not something I wanted to deal with in an intro to programming class.

**Comment by scerruti** [comment]
I am updating this since it is still happening and I don't have a fix or workaround.

https://miracosta.instructuremedia.com/embed/b8cc0916-aa90-497a-8e47-f7c555d05768
